src/uct/ib/ud/base/ud_ep.c

```diff
@@ -1011,7 +1011,12 @@ static void uct_ud_ep_do_pending_ctl(uct_ud_ep_t *ep, uct_ud_iface_t *iface)
         skb =  uct_ud_ep_resend(ep);
     } else if (uct_ud_ep_ctl_op_check(ep, UCT_UD_EP_OP_ACK)) {
         if (uct_ud_ep_is_connected(ep)) {
-            skb = ucs_unaligned_ptr(&iface->tx.skb_inl.super);
+            if (iface->config.max_inline) {
```

Comment:
what if iface->config.max_inline==1

Reply:
technically it will fail, but are there such devices?

Reply:
@yosefe fixed

Reply:
need to check if inline size is >= minimum, not >0 ...

Reply:
right, fixed